_conferences/blrdroiddevfest-2017.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "GDG BlrDroid DevFest"
```

Comment:
Since dev fests are a standard thing we name them all the same. Can you make this just "GDG DevFest"?